telegram/_bot.py

```diff
@@ -363,11 +363,24 @@ def __deepcopy__(self, memodict: Dict[int, object]) -> NoReturn:
         raise TypeError("Bot objects cannot be deepcopied!")
 
     def __eq__(self, other: object) -> bool:
+        """Defines equality condition for the :class:`telegram.Bot` object.
+        Two objects of this class are considered to be equal if their all parameters
+        are identical.
+
+        Returns:
+            :obj:`True` if both objects have all parameters identical. :obj:`False` otherwise.
```

Comment:
```suggestion
        Two objects of this class are considered to be equal if their  attributes :attr:`bot`
        are equal.

        Returns:
            :obj:`True` if both attributes :attr:`bot` are equal. :obj:`False` otherwise.
```